src/test/java/com/axone/devintest/tests/AdditionTest3.java

```diff
@@ -0,0 +1,15 @@
+package com.axone.devintest.tests;
+
+import com.axone.devintest.calc.Addition;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AdditionTest3 {
+
+    @Test
+    public void add(){
+        Addition a = new Addition();
```

Comment:
@ranjithkumarputta Why you have created object for a class at medthod level instead you can create an object at class level so that same object can use in all other remaining test cases